packages/@react-spectrum/table/src/table.css

```diff
@@ -27,6 +27,12 @@
   height: 100%;
 }
 
+.react-spectrum-Table-cellWrapper:last-child {
```

Comment:
does this work in RTL languages as well? you may need to use inset-inline-start and inset-inline-end
do we need to unset left? it shouldn't matter if the left and right are both set for the last cell i think

Reply:
I'm not seeing this bug at all for RTL. Let me know if this should be changed anyway.